### ISSUE TITLE
Add immunity for suffocation for ~30 seconds after transitioning shards

### ISSUE THREAD

**Issue by Diet-Cola** [DevotedMC/Artemis#15]
This will make it so people can cross shards without worry of dying by the time their screen loads on the other side.

**Comment by caucow** [comment]
aren't you able to see the shard on the other side before transitioning? ie "don't cross a shard directly into a wall"

also, you can use KeepAlives with a unique id to synchronize with the client so once the chunks have been sent you can send a keepalive packet then wait for a keepalive response with the same id or a move packet/event moving into a different block to cancel the immunity

**Comment by Diet-Cola** [comment]
Yes however there are some situations where players can be mining from Shard A -> B and not even realise the border is there (If you don't use the vanilla WB). This also ties a seperate issue in too which some border blocks cannot be broken because they are assumed to be outside the border
